install_scripts/templates/common/kubernetes.sh

```diff
@@ -551,6 +551,19 @@ k8sNamespaceExists() {
     kubectl get namespaces | grep "$1" > /dev/null
 }
 
+#######################################
+# Return status code 0 if there is a storageclass with the default annotation, else 1
+# Globals:
+#   None
+# Arguments:
+#   provisioner
+# Returns:
+#   None
+#######################################
+defaultStorageClassExists() {
+    kubectl get storageclass -o=jsonpath='{.items[*].metadata.annotations}' | grep -q "storageclass.kubernetes.io/is-default-class":"true"
```

Comment:
does that argument to grep get treated as a single argument because there isn't a space, even though there are sets of double quotes?

Reply:
yes 👍